zollege_lti/processors.py

```diff
@@ -16,7 +16,10 @@ def personal_user_info(xblock):
     names_list = user_full_name.split(' ', 1)
 
     params = {
+        'lis_person_contact_email_primary': user.email,
```

Comment:
is not the email sent by default? or is it because you need to send it in the `lis_person_contact_email_primary` key?

Reply:
Yes, They need the lis_person_name_given param.